included_software.md

```diff
@@ -9,19 +9,21 @@ For instructions on how to run this image locally to test your build, please see
 The specific patch versions included will depend on when the image was last built (except Ruby). It is highly suggested you depend only on minor versions, so that we can ensure the language has the latest updates (especially if security related).
 
 * Ruby - `RUBY_VERSION`, `.ruby-version`
-  * 2.6.2 (default)
-  * 2.7.2
+  * 2.7 (default)
+  * 2.6
```

Comment:
can we be more explicit about which 2.6 and 2.7 - for ruby it kinda matters.

Reply:
Sure, I'll specify which versions in the document.